src/python/BuildingControlsSimulator/DataClients/DataClient.py

```diff
@@ -307,6 +307,19 @@ def get_data(self):
             simulation_epw_dir=self.simulation_epw_dir,
         )
 
+        #add nsrdb solar data fields
+        fill_nsrdb_data = self.weather.get_nsrdb(
+            lat=self.sim_config["latitude"], 
+            long=self.sim_config["longitude"], 
+            year=self.sim_config["start_utc"].year
+        )
+        self.weather.fill_nsrdb(
```

Comment:
this call can be done within `self.weather.get_nsrdb()` at the end of the function, and it then does not need to return anything similar to `self.sensors.drop_unused_room_sensors()` called above.

Reply:
sure.  this will be very streamlined now